generators/create-react-app/templates/base/package.json.ejs

```diff
@@ -13,7 +13,7 @@
     "@types/react-dom": "^16.9.8",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "react-scripts": "4.0.1",
+    "react-scripts": "4.0.3",
```

Comment:
```suggestion
"react-scripts": "^4.0.3"
```
As described in this [issue](https://github.com/thetribeio/generator-project/issues/293), react-scripts 4.0.3 has still dependencies issue with `@babel/core` when using npm 7.
It should be resolved in a future release, so is it possible to be more permissive about minor version ?

Reply:
When you generate a project with create-react-app, the version of react-scripts is fixed on the last version like this. I prefer to not differ from the CRA generated template when possible and I think this is the case here, I will just have to update react-scripts again when they release a new version.